DQMOffline/MuonDPG/plugins/plugins.cc

```diff
@@ -0,0 +1,21 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include <DQMOffline/MuonDPG/src/DTTnPEfficiencyTask.h>
+DEFINE_FWK_MODULE(DTTnPEfficiencyTask);
+
+#include <DQMOffline/MuonDPG/src/CSCTnPEfficiencyTask.h>
+DEFINE_FWK_MODULE(CSCTnPEfficiencyTask);
+
+#include <DQMOffline/MuonDPG/src/RPCTnPEfficiencyTask.h>
+DEFINE_FWK_MODULE(RPCTnPEfficiencyTask);
+
+#include "DQMOffline/MuonDPG/plugins/TnPEfficiencyClient.h"
+DEFINE_FWK_MODULE(TnPEfficiencyClient);
+
+//#include "DQMOffline/MuonDPG/src/TnPEfficiencyClient.h"
```

Comment:
... and in any case these commented out lines are not needed and should be removed

Reply:
Removing `plugins.cc`